CRM/CiviAwardsPaymentsTab/Hook/AddCiviCaseDependentAngularModules/AddModule.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use CRM_CiviAwards_Service_FinanceManagementSetting as FinanceManagementSettingService;
+
+/**
+ * Add Payments Tab module.
+ *
+ * This will add the payments tab module to awards instances that support
+ * finances.
+ */
+class CRM_CiviAwardsPaymentsTab_Hook_AddCiviCaseDependentAngularModules_AddModule {
```

Comment:
The class name seems a bit ambiguous, why don't we use rename to `PaymentsTabModule`. I removed the word `Add` because the folder name already describes that classes in this folder are adding to the civicase angular dependent modules.